Header header
int32 robot_id
float64 x_goal
float64 y_goal